data/ai/lua/ca_recruit_rushers.lua

```diff
@@ -16,7 +30,8 @@ local params = {
             return true
```

Comment:
This still needs to be
return not ai.aspects.passive_leader
since castle_switch returns 0 if the leader is passive, but that means that normally the leader is available for village capturing and should be true, except for the case where the leader never moves (passive), where it should be false, thus the need for the extra condition.

That should also become part of the default behaviour of the recruit engine when leader_takes_village is unset.

Reply:
The point of the parameter is for recruitment to decide if it needs to recruit units to capture every village near the current castle.
If the leader might capture one, then the value is true, if the leader definitely won't then it's false. If the recruit engine is not told which, it currently assumes true, but that isn't accurate for a passive leader.

Reply:
I think this is taken care of now.  I split it up into two commits: 0bcb98cf and b92d8e8

The () I added around the old conditional in the latter commit are unnecessary for the castle_switch CA, but that way it will also work with other functions which fail to check for passive_leader internally.